test/clients/whatsapp_test.exs

```diff
@@ -28,7 +32,8 @@ defmodule Companion.Clients.WhatsappTest do
         url: "https://whatsapp/v1/messages",
         headers: [
           {"content-type", "application/json"},
-          {"authorization", "Bearer token"}
+          {"authorization", "Bearer token"},
+          {"user-agent", "nurseconnect-companion"}
```

Comment:
This seems like something you may want to factor out into a `standard_headers` function that you can use in all the places it needs to be used.